web/src/components/organisms/Settings/Members/index.tsx

```diff
@@ -148,6 +148,7 @@ const Members: React.FC = () => {
 const PaddedContent = styled(Content)`
   margin: 16px;
   background-color: #fff;
+  min-height: 100%;
```

Comment:
Same here. If min-height is needed, you can disregard this comment.

Reply:
@KaWaite , yes it is
Figma requires to have the center content to be white and full height in certain pages

Reply:
done.

Reply:
actually, min-height is required, I reverted to it after testing.
yes, content can be more than 100%.